query/src/main/java/org/infinispan/query/clustered/QueryResponse.java

```diff
@@ -45,6 +45,17 @@ public QueryResponse(NodeTopDocs nodeTopDocs, UUID nodeUUid, int resultSize) {
       this.resultSize = resultSize;
    }
 
+   public boolean nonEmpty() {
```

Comment:
@anistor both ways are fine to me, I was using ```getResultSize``` and @karesti suggested an extra method. Since this class is used internally only, let's keep it